src/test/resources/features/comp/COMPSearch.feature

```diff
@@ -25,7 +25,7 @@ Feature: COMP Search
   @COMPRegression
   Scenario: User can search for a COMP case by its case reference
     When I create a single "COMP" case
-    And I search for the COMP case by its case reference
+    And I search for the "COMP" case by its case reference
```

Comment:
seeing as this doesnt ues the casetype variable, maybe it should just read 'And I search for the case by its case reference'

Reply:
Resolved